src/main/groovy/org/octopusden/octopus/license/management/plugins/gradle/tasks/LicensedDependenciesAnalyzingTask.groovy

```diff
@@ -144,35 +145,24 @@ class LicensedDependenciesAnalyzingTask extends DefaultTask {
                 supportedGroups.addAll(supportedGroupsString.split(",").collect { it.trim() })
             }
 
-            if (project.hasProperty(CRS_URL) && isValidUrl(CRS_URL)) {
-                def url = new URL("${project.property(CRS_URL)}/rest/api/2/common/supported-groups")
-                def connection = (HttpURLConnection) url.openConnection()
-                connection.requestMethod = 'GET'
-
-                logger.info("Requesting supported groups from URL: ${url}")
-
-                def responseCode = connection.getResponseCode()
-
-                if (responseCode == HttpURLConnection.HTTP_OK) {
-                    def inputStream = connection.getInputStream()
-                    def responseText = inputStream.text
-
-                    def jsonSlurper = new JsonSlurper()
-                    def supportedGroupsRes = jsonSlurper.parseText(responseText)
-
-                    if (supportedGroupsRes instanceof List && supportedGroupsRes.every{ it instanceof String }) {
-                        logger.info("Successfully get supported groups from component registry: ${supportedGroupsRes}")
-                        supportedGroups.addAll(supportedGroupsRes)
-                    } else {
-                        logger.error("Unexpected format of supported groups: ${supportedGroupsRes}")
+            def componentsRegistryApiUrl = project.hasProperty(CRS_URL) ? project.findProperty(CRS_URL).toString() : null
```

Comment:
Check that only one of componentsRegistryURL and supportedGroupIds are specified; fail otherwise